packages/manager/src/components/Paper.tsx

```diff
@@ -0,0 +1,42 @@
+import _Paper, { PaperProps } from '@mui/material/Paper';
+import { styled } from '@mui/material/styles';
+import * as React from 'react';
+
+import FormHelperText from './core/FormHelperText';
+
+const StyledPaper = styled(_Paper)<Props>(({ theme, ...props }) => ({
+  borderColor: props.error ? `#ca0813` : undefined,
```

Comment:
```suggestion
  borderColor: props.error ? theme.color.red : undefined,
```

Can we avoid a hardcoded value for the color here?